### DISCUSSION TITLE
Setting Path Params in context

### DISCUSSION THREAD

**Discussion by Winkingroad** [gofr-dev/gofr#148] [Q&A] [answered]
I cannot set PathParams in my gofr context when creating a context for my Unit Testing. It always gives an issue that no SetPathParams property exists in gofr context.

**Comment by Umang01-hash** [comment, marked as answer]
@Winkingroad setting path params in context can be done using `SetPathParams` method of context.
Here is an example where we are using the same in unit testing: https://github.com/gofr-dev/gofr/blob/fe31a5da35809877cc7ef663dbc7b0e10fe0e7ba/examples/using-postgres/handler/handlers_test.go#L125

Feel free to refer to it.
